Cargo.toml

```diff
@@ -12,11 +12,11 @@ description = "A command-line utility for easily compressing and decompressing f
 
 [dependencies]
 atty = "0.2.14"
-bstr = { version = "1.8.0", default-features = false, features = ["std"] }
+bstr = { version = "1.6.2", default-features = false, features = ["std"] }
 bzip2 = "0.4.4"
-clap = { version = "4.4.8", features = ["derive", "env"] }
+clap = { version = "4.4.5", features = ["derive", "env"] }
 filetime = "0.2.22"
-flate2 = { version = "1.0.28", default-features = false }
+flate2 = { version = "1.0.27", default-features = false }
```

Comment:
I did a last re-review, and found this which is unusual, I'm assuming it happened when you solved some merge conflict.

Before we merge, can you revert these?

Reply:
Oops, must have slipped unnoticed during the rebase.
Should be ok now.